lib/resources/r4/appointment_patch.yaml

```diff
@@ -88,3 +88,77 @@ operations:
         <li>This patch operation must be accompanied by all other video visit link patch operations</li>
         <li>The value for this patch operation will be applied to both the provider video visit link and the patient video visit link</li>
       </ul>
+  - name: replace-provider-vmr
+    path: /contained/0/telecom/0/value
+    operation: replace
+    type: string
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#ContactPoint.value
+    description: The provider video visit link
+    example: |
+      {
+        "op": "replace",
+        "path": "/contained/0/telecom/0/value",
+        "value":  "http://providerlink.vmr.net"
+      }
+    note: >
+      <ul>
+        <li>This patch operation may be used only for video visit appointments</li>
+        <li>This patch operation must be accompanied by all other video visit link patch operations</li>
+        <li>The appointment must already contain VMR data</li>
```

Comment:
This isn't true right? a replace on a video visit appointment with no links should act as an add?

Reply:
Ahh you're right, I'll remove that note

Reply:
removed from each https://github.com/cerner/fhir.cerner.com/pull/385/commits/ab0f399f4c56a6788c04ccd244ad0a9e9453ede9#diff-07dbf4d3264a1e7a98f8d854b77f88e9L125